lib/t/news.ex

```diff
@@ -216,4 +228,12 @@ defmodule T.News do
   defp last_seen_id(user_id) do
     SeenNews |> where(user_id: ^user_id) |> select([n], n.last_id) |> Repo.one()
   end
+
+  defp user_inserted_at(user_id) do
```

Comment:
`user_id` already contains timestamp

Reply:
@ruslandoga what could decoding look like?

Reply:
extract the first 64 bits, that's a unix timestamp in ms, turn it into a datetime using from_unix function